app/res/xml/preferences_developer.xml

```diff
@@ -130,5 +130,10 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-bulk-performance"
         android:title="Enable cutting-edge performance improvements"/>
-
+    <ListPreference
```

Comment:
@amstone326 ok, validated that the project has the flag set correctly in their app preferences, so this looks good to go. Good catch.